BenadrylAutomation/src/main/java/com/jnj/pageobjects/ComparePage.java

```diff
@@ -197,7 +197,7 @@ public void verifyRelatedProducts(String heading, String product, String expecte
 			Action.explicitWait(buyNowBtn(product), 30);
 			buyNowBtn(product).click();
 			extentInfoLog("Clicked on : ", "buy now button");
-			signup.closeSignUpPopup();
+			Action.waitFor(2000);
```

Comment:
Why is a static sleep placed here ?
@RashiTiwari21

Reply:
@RashiTiwari21 removed this static wait and added signup.closeSignUpPopup();